docs/source/operations/pipeline.rst

```diff
@@ -127,7 +127,12 @@ Optional
 
 .. option:: +inv
 
-    Invert a step in a pipeline.
+    Invert a step in a pipeline. Note, cct handles more situations:
+
+::
+
+    echo -88 42    |proj +zone=16 +proj=pipeline +step +proj=utm +step +proj=utm +inv #fails
+    echo -88 42 0 0|cct  +zone=16 +proj=pipeline +step +proj=utm +step +proj=utm +inv #succeeds
```

Comment:
I've removed the comment about "cct handles more situations" as I don't think it belongs there, and the example with "proj" that fails as it's going to create more confusion

```suggestion
    Invert a step in a pipeline.

    ::

        echo -88 42 0 0|cct  +zone=16 +proj=pipeline +step +proj=utm +step +proj=utm +inv
```